packages/cli/generation/ir-generator/src/dynamic-snippets/DynamicSnippetsConverter.ts

```diff
@@ -140,7 +140,17 @@ export class DynamicSnippetsConverter {
             declaration: this.convertDeclaration({ name: endpoint.name, fernFilepath: endpoint.fernFilepath }),
             location,
             request: this.convertRequest({ endpoint }),
-            response: DynamicSnippets.Response.json(),
+            response: (endpoint.response?.body?._visit<DynamicSnippets.Response>({
```

Comment:
is it safe to go ahead and produce this at this point? Do our tests ensure that the consumers are graceful?

Reply:
It should be fine.